packages/SwingSet/test/vat-admin/test-innerVat.js

```diff
@@ -25,7 +23,7 @@ async function testVatCreationFromBuild(t, withSES) {
   t.end();
 }
 
-test('VatAdmin inner vat creation', async t => {
+test.skip('VatAdmin inner vat creation', async t => {
```

Comment:
can't really land this until all the previous tests are passing, we don't want to let vat-admin regress

Reply:
Agreed.  I'll wait for your new SES to land before I try to work on this PR further.